library/src/main/java/io/nlopez/clusterer/Clusterer.java

```diff
@@ -114,8 +124,10 @@ public boolean onMarkerClick(Marker marker) {
             if (cluster != null) {
                 CameraUpdate update = CameraUpdateFactory.newLatLngBounds(cluster.getBounds(), CLUSTER_CENTER_PADDING);
                 googleMap.animateCamera(update, CAMERA_ANIMATION_DURATION, null);
+                listener.clusterClicked(cluster);
```

Comment:
The listener should be null checked in here, otherwise it will crash with a NPE if not set.